browser/src/control/jsdialog/Widget.TreeView.js

```diff
@@ -426,15 +434,24 @@ function _handleKeyEvent(event, listElements, builder, data) {
 	if (event.key === 'ArrowDown') {
 		if (currIndex < 0)
 			_changeFocusedRow(listElements, currIndex, 0);
-		else if (currIndex < treeLength - 1)
-			_changeFocusedRow(listElements, currIndex, currIndex + 1);
-
+		else {
+			var nextIndex = currIndex + 1;
+			while (currIndex < treeLength - 1 && listElements[nextIndex].clientHeight <= 0)
```

Comment:
currIndex or nextIndex?  (in the first condition)

Reply:
thx, you are right :)